javaScriptNumbers-MathObject/codeChallenge/random.js

```diff
@@ -0,0 +1,7 @@
+const getInput = prompt('Please input a Number:');
+
+const inputInt = parseInt(getInput);
+
+const randomNumber = Math.floor(Math.random() * inputInt) + 1;
+
+console.log(`${randomNumber} is a random number between 1 and ${inputInt}` );
```

Comment:
Another good challenge here would be to have a fallback message in case the user didn't enter a valid number. For example, if the input equals a string such as "Lorem Ipsum", the message should say "Invalid input. Please input a number: ".

Reply:
Noted on the challenge. Added code updates

ae0eca10e63c845622502a9b26fd37322ded7d47